### DISCUSSION TITLE
WorldServer Cannot Find Maps

### DISCUSSION THREAD

**Discussion by Quest79** [azerothcore/azerothcore-wotlk#5580] [General]
Hi, Ive been looking at this for two days now and cannot find an answer that fixed this and Im very confused.

- [Guide being used is here](https://www.azerothcore.org/wiki/Install-with-Docker)
- I followed each step precisely as well as forcing ownership to the top level `azerothcore-wotlk` folder
- When i get to step **`3) Run the containers`** everything runs fine except the world server giving me the below error (see pic link).
- I have change the worldserver.conf file to point directly to where the maps folder is, to the folder that contains the map folder and a dozen combinations in between. 
- I am confused why it cannot see the maps, and why the maps are not in a data folder where it was pointed to by default? There is no data folder there, and the data folder I do have has no map files in it. Do I need to drag the map files to there? Why was this not part of the setup ifso?
- You can also see in the below image that worldserver.conf is being read `"Using configuration file /azerothcore/env/dist/etc/worldserver.conf"` yet the file is in `azerothcore-wotlk\env\docker\etc` can this be explained to me? 

<img width="1461" alt="ZEN2 050121 102228 2922x1234" src="https://user-images.githubusercontent.com/635094/116790728-b8495100-aa6a-11eb-8a25-60b157cafba8.png">

Thank you!

**Comment by Kitzunu** [comment]
Bad extraction

**Comment by Quest79** [comment]
What does that mean? What should the extraction look like? I got no errors during any of the steps I did.

**Comment by FrancescoBorzi** [comment]
hello @Quest79 and welcome. GitHub is not the right place to ask for support, please read:

https://www.azerothcore.org/wiki/How-to-ask-for-help

**Reply by Quest79** [reply]
Thanks, I did make an effort to find the proper way to ask for help, I obviously missed that article somehow. I will post in SO

**Comment by mrs-th** [comment]
Hi, I had exactly the same error. In may case it was a typo in my DataDir in the worldserver.conf. After fixing the typo, the worldserver started without issues. Maybe it's the same for you.
